src/scripts/html-transforms.js

```diff
@@ -27,6 +47,21 @@ export function removeStyleAndScript (document) {
 
   return document;
 }
+ 
+/**
+ * Creates a transform that prevents navigation from within a diff. This is 
+ * needed so that diffs can still be compared correctly. Function forces
+ * links to open in a new tab when clicked instead of inside the iframe.
+ * @param {HTMLDocument} document The html document to transform.
+ * @returns {HTMLDocument}
+ */
+// Add target="_blank" to all <a>tags
```

Comment:
Hey just one tiny nit: can you move this comment down to be right above `document.querySelectorAll('a')`? Or you could just delete it. Thanks!

Reply:
Got it 
✅